target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -160,6 +161,11 @@ ipq40xx_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
+	mikrotik,hap-ac2)
+		wan_mac=$(cat /sys/firmware/mikrotik/hard_config/mac_base)
+		lan_mac=$(macaddr_add $wan_mac 1)
+		label_mac="$wan_mac"
+		;;
```

Comment:
Since you're adding a subtarget, this belongs to the subtarget environment. I'd recommend following the style found in target/linux/ath79/mikrotik/base-files/etc/board.d/02_network so that new mikrotik boards can easily be added